tests/caasp/stack_bootstrap.pm

```diff
@@ -35,6 +35,19 @@ sub select_nodes {
     wait_still_screen 2;
 }
 
+# 10% of clicks are lost because of ajax refreshing Velum during click
```

Comment:
I don't have enough insights, but why not to do normal clicks or assert_and_click till you match needle with checked checkbox? I guess you also can come up with the feature request for openQA framework to have this coordinate calculations there and not in the test. WDYT?

Reply:
Long story behind :)

First part selects "master.openqa.test" node:
 - in this case you don't know where to assert_and_click (which line).

Second part selects 2 more random nodes. You can't just assert_and_click because:
 - if you do it during ajax refresh it would be lost (90% success rate)
 - until_needlematch would always match because of node selected in first part
 - that is what I am doing now and it does not work (see "fix for")

I don't think there is need for it in openQA framework (code is too specific & you need custom needles for it).

Reply:
Thanks for clarification. I see the challenge here and wonder how we can improve it, e.g. by wrapping webdriver in openQA. As we have researched similar things for yast ruby gui's, so we can avoid coordinates magic and changing fonts issues =). So in general I don't like what we do here, but I see no alternatives as of now.